recipes/neuraxle/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "neuraxle" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8dc696bf67f9b45be778eb2703221525aed94534da704a79ced3e531ffef5fce
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.6
+    - numpy >=1.16.2
+    - scipy >=1.4.1
+    - scikit-learn >=0.24.1
+    - matplotlib ==3.3.4
+    - joblib >=0.13.2
+    - Flask >=1.1.4
+    - Flask-RESTful >=0.3.7
+
+test:
+  requires:
+    - pytest
+  source_files:
+    - testing
+  commands:
+    - pytest testing --ignore=testing/examples --ignore=testing/metaopt/test_tpe.py
+
+about:
+  home: https://www.neuraxle.org/stable/index.html
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'A Sklearn-like Framework for Hyperparameter Tuning in Deep Learning projects. Finally have the right abstractions and design patterns to properly do AutoML. Let your pipeline steps have hyperparameter spaces. Enable checkpoints to cut duplicate calculations. Go from research to production environment easily.'
```

Taken from [the `description`]( https://github.com/Neuraxio/Neuraxle/blob/0615701b781c948e4ec38fa61c6b3a5d8d72c147/setup.py#L33-L34 ) and wrapped:

```suggestion
  summary: >-
    Neuraxle is a Machine Learning (ML) library for building neat pipelines, providing the right
    abstractions to both ease research, development, and deployment of your ML applications.
```